NOTICE

```diff
@@ -0,0 +1,7 @@
+Ensembl eHive
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2024] EMBL-European Bioinformatics Institute
+
+This product includes software developed at:
+- EMBL-European Bioinformatics Institute
+- Wellcome Trust Sanger Institute
```

Comment:
For some reasons I cannot see Genentech here, although I can see them mentioned in the NOTICE file by looking into the branch. Just let's make sure the NOTICE file is correct. Thx